modules/Microsoft.HealthDataServices/fhirservices/deploy.bicep

```diff
@@ -0,0 +1,298 @@
+@description('Required. The name of the FHIR service.')
+@maxLength(50)
+param name string
+
+@allowed([
+  'fhir-R4'
+  'fhir-Stu3'
+])
+param kind string = 'fhir-R4'
+
+@description('Required. The name of the parent health data services workspace.')
+param workspaceName string
+
+@description('Optional. List of Azure AD object IDs (User or Apps) that is allowed access to the FHIR service.')
+param accessPolicyObjectIds array = []
+
+@description('Optional. The list of the Azure container registry login servers.')
+param acrLoginServers array = []
+
+/*
+{
+  digest: 'string'
+  imageName: 'string'
+  loginServer: 'string'
+}
+*/
```

Comment:
Leftover? 

If not - it is instead intended to show how the object looks like, we'd recommend to add it as an additional 'Parameter Usage: `acrOciArtifacts`' entry to the module's readme. ([ref](https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.AnalysisServices/servers/readme.md#parameter-usage-tags))

Reply:
makes sense, I will move that to the readme